scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -313,6 +313,13 @@ object Decl {
       tparams: List[scala.meta.Type.Param],
       bounds: scala.meta.Type.Bounds
   ) extends Decl with Member.Type
+  @ast class Given(
+      mods: List[Mod],
+      name: scala.meta.Name,
```

Comment:
use Term.Name if name cannot be Anonymous

Reply:
or actually Type.Name here

Reply:
Good catch! Though I think givens should actually be `Term.Name` the same as defs, vals and vars. `Type.Name` is for more class like entities.

Changed here to TermName and fixed to use it always.